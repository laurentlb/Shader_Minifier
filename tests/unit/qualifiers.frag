#version 130

in vec3 foo;

float f(in float a) {
  return a;
}

float g(const in float a) {
  return a;
}

float h(inout float a) {
  return a;
}
