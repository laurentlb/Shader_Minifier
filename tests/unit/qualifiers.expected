/* File generated with Shader Minifier 1.2
 * http://www.ctrl-alt-test.fr
 */
#ifndef QUALIFIERS_EXPECTED_
# define QUALIFIERS_EXPECTED_
# define VAR_foo "f"

const char *qualifiers_frag =
 "in vec3 f;"
 "float v(float f)"
 "{"
   "return f;"
 "}"
 "float t(const float f)"
 "{"
   "return f;"
 "}"
 "float o(inout float f)"
 "{"
   "return f;"
 "}";

#endif // QUALIFIERS_EXPECTED_
